.goreleaser.yaml

```
@@ -0,0 +1,196 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema-pro.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 2
+pro: true
+project_name: thor
+
+metadata:
+  license: "LGPL-3.0"
+  homepage: "https://github.com/{{ .Env.REPOSITORY }}"
+  description: "A general purpose blockchain highly compatible with Ethereum's ecosystem"
+  full_description: "Thor is the official Golang client for VeChainThor, the public blockchain powering the VeChain ecosystem. VeChainThor is designed for real-world adoption, enabling scalable, low-cost, and sustainable applications."
+
+partial:
+  by: target
+
+builds:
+  - id: "thor"
+    main: ./cmd/thor
+    binary: thor
+    ldflags:
+      - -s -w
+      - -X main.version={{.Env.THOR_VERSION}}
+      - -X main.gitCommit={{.ShortCommit}}
+      - -X main.gitTag={{.Tag}}
+      - -X main.copyrightYear={{.Env.COPYRIGHT_YEAR}}
+    env:
+      - CGO_ENABLED=1
+  - id: "disco"
+    main: ./cmd/disco
+    binary: disco
+    ldflags:
+      - -s -w
+      - -X main.version={{.Env.DISCO_VERSION}}
+      - -X main.gitCommit={{.ShortCommit}}
+      - -X main.gitTag={{.Tag}}
+      - -X main.copyrightYear={{.Env.COPYRIGHT_YEAR}}
+    env:
+      - CGO_ENABLED=1
+
+signs:
+  - cmd: gpg
+    args:
+    - "--batch"
+    - "-u"
+    - "{{ .Env.GPG_FINGERPRINT }}"
+    - "--output"
+    - "${signature}"
+    - "--detach-sign"
+    - "${artifact}"
+
+binary_signs:
+  - cmd: gpg
+    args:
+      - "--batch"
+      - "-u"
+      - "{{ .Env.GPG_FINGERPRINT }}"
+      - "--output"
+      - "${signature}"
+      - "--detach-sign"
+      - "${artifact}"
+    output: true
+
+docker_signs:
+  - artifacts: all
+    stdin: "{{ .Env.COSIGN_PASSWORD }}"
+    output: true
+
+archives:
+  - formats: [ 'tar.gz' ]
+    id: thor
+    ids:
+      - thor
+    format_overrides:
+      - goos: windows
+        formats: [ 'zip' ]
+    name_template: 'thor-{{ .Tag }}-{{ .Os }}-{{ .Arch }}'
+    files:
+      - LICENSE
+      - README.md
+  - formats: [ 'tar.gz' ]
+    id: disco
+    ids:
+      - disco
+    format_overrides:
+      - goos: windows
+        formats: [ 'zip' ]
+    name_template: 'disco-{{ .Env.DISCO_VERSION }}-{{ .Os }}-{{ .Arch }}'
+    files:
+      - LICENSE
+      - README.md
+
+snapshot:
+  version_template: SNAPSHOT-{{ .Commit }}
+
+checksum:
+  disable: false
+  name_template: 'checksums.txt'
+
+release:
+  skip_upload: false
+  github:
+    owner: vechain
+    name: thor
+  ids:
+    - disco
+    - thor
+  mode: append
+  prerelease: '{{ .IsNightly }}true{{ else }}false{{ end }}'
```

Is this missing an if clause ?
```suggestion
  prerelease: '{{ if .IsNightly }}true{{ else }}false{{ end }}'
```